tiatoolbox/utils/misc.py

```diff
@@ -335,6 +337,59 @@ def contrast_enhancer(img, low_p=2, high_p=98):
     return np.uint8(img_out)
 
 
+def read_point_annotations(input_table):
+    """Read annotations as pandas DataFrame.
+
+    Args:
+        input_table (str or pathlib.Path or ndarray or pd.DataFrame): path to csv, npy
+         or json or an ndarray. first column in the table represents x position, second
+         column represents y position. The third column represents the class. If the
+         table has headers, the header should be x, y & class.
+
+    Returns:
+        pd.DataFrame: DataFrame with x, y location and class type.
+
+    Examples:
+        >>> from tiatoolbox.utils.misc import read_point_annotations
+        >>> labels = read_point_annotations('./annotations.csv')
+
+    """
+    if isinstance(input_table, (str, pathlib.Path)):
+        _, _, suffix = split_path_name_ext(input_table)
+
+        if suffix == ".npy":
+            out_table = np.load(input_table)
+            if out_table.shape[1] == 2:
+                out_table = pd.DataFrame(out_table, columns=["x", "y"])
+                out_table["class"] = None
+            else:
+                out_table = pd.DataFrame(out_table, columns=["x", "y", "class"])
+
+        elif suffix == ".csv":
+            out_table = pd.read_csv(input_table)
+            if "x" not in out_table.columns:
```

Comment:
what if csv file has header? 
I guess we can force the user to use a csv file without header, but it is nicer to consider cases that have a first-line header.

Reply:
the header should be `x`, `y` and `class` as in the documentation.

Reply:
Sorry, I get it wrong. this part of code supports both header and headerless csv files. But, the two column situation is not covered here as well.  I can work on it if you want.

Reply:
Re the csv, same as above, error will likely occur here due to wrong csv format and we would not be able to deal with it. I think we should wrap it in a `try` block and only accept strict `csv` format and mention it so within the doc.